src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -1030,6 +1001,75 @@ private bool TryGetIncludeCtes(string resourceType, out List<string> ctes)
             return _includeLimitCtesByResourceType.TryGetValue(resourceType, out ctes);
         }
 
+        private bool IsSortValueNeeded(SearchOptions context)
+        {
+            if (context.Sort.Count == 0)
+            {
+                return false;
+            }
+
+            var sortParamInfo = context.Sort[0].searchParameterInfo;
```

Comment:
Why do we check only position 0 of this collection?  Is it no possible to have more than one entry?

Reply:
We used to only support one parameter to sort on. The code now includes support for a specific 2-parameter sort. I have updated the code to handle that.